src/main/java/ay2021s1_cs2103_w16_3/finesse/ui/tabs/UserGuideTabPane.java

```diff
@@ -59,6 +59,12 @@ private void initializeUserGuide() {
 
         WebEngine webEngine = webView.getEngine();
 
+        // Override fonts to call the default macOS system font without using '-apple-system'.
+        webEngine.setUserStyleSheetLocation(getClass().getResource("/view/WebView.css").toString());
+
+        // Log user agent.
+        logger.info("User Agent: " + webEngine.getUserAgent());
```

Comment:
Pre-emptive logging so that any bugs reported in the future can be investigated.